Dockerfile

```diff
@@ -119,11 +120,13 @@ FROM base-nginx AS build-nginx
 COPY deploy/config/php-fpm.conf /etc/nginx/php-fpm.conf
 COPY deploy/config/server.conf /etc/nginx/conf.d/default.conf
 
+# WordPress view bootstrapper
+COPY public/index.php /var/www/html/public/index.php
```

Comment:
Methodology: 5. Since we're not doing COPY . . , copy index.php from host instead.